src/graphics/model/model_output.cpp

```diff
@@ -328,7 +328,10 @@ void ModelOutput::WriteOldModel(const CModel& model, std::ostream &stream)
         t.material.ambient = triangle.ambient;
         t.material.diffuse = triangle.diffuse;
         t.material.specular = triangle.specular;
-        strncpy(t.texName, triangle.tex1Name.c_str(), 20);
+
+        strncpy(t.texName, triangle.tex1Name.c_str(), sizeof(t.texName)-1);
```

Comment:
Linter complains here about `Implicit cast bool -> 'char'`.

Reply:
This is line 332. The warning is for line 322 (10 lines above this one), for this little bit:
```
    t.used = true;
```

Reply:
Oh, my bad.